src/main/java/com/fiap/intelligrid/controller/EnderecoController.java

```diff
@@ -0,0 +1,71 @@
+package com.fiap.intelligrid.controller;
+
+import com.fiap.intelligrid.domain.exception.DefaultException;
+import com.fiap.intelligrid.domain.request.EnderecoRequest;
+import com.fiap.intelligrid.domain.response.EnderecoResponse;
+import com.fiap.intelligrid.service.EnderecoService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/endereco")
+@RestController
+public class EnderecoController {
+
+	private final EnderecoService enderecoService;
+
+	private EnderecoController(EnderecoService enderecoService) {
+		this.enderecoService = enderecoService;
+	}
+
+	@GetMapping("/cep/{cep}")
```

Comment:
Do jeito que está você consegue dar um post e salvar um endereço, porém você não consegue recuperar esse endereço, pois só tem a busca passando um cep real e utilizando a api do viacep, mas a que vc salvou fica perdida.

Reply:
Eu estava esperando o teu commit pra criar um buscar por pessoa.
Ve se faz sentido, bele?
Eu pensei que quando a gnt for exibir o endereço, vai ser pra pessoa já logada, ou seja, a gnt vai buscar por pessoa e nunca por cep ou id do endereço.
O que vc acha?